src/views/Welcome.vue

```diff
@@ -5,26 +5,26 @@
         <div class="d-flex flex-no-wrap justify-space-around align-center">
           <div class="ml-3"><v-progress-circular size="50" indeterminate color="primary"></v-progress-circular></div> 
           <div>
-            <v-card-text>මේ මෘදුකාංගයේ දත්ත තවමත් copy වෙමින් පවතී. මොහොතක් ඉවසන්න. මෙය පළමු ස්ථාපනය කිරීමේදී පමණක් සිදුවේ.</v-card-text>
+            <v-card-text>මෙම මෘදුකාංගයේ දත්ත පිටපත් වෙමින් පවතී. මොහොතක් රැඳී සිටින්න. මෙය පළමු ස්ථාපනයේ දී පමණක් සිදුවේ.</v-card-text>
           </div>
         </div>
       </v-card>
     </v-dialog>
 
     <v-banner icon="mdi-alert-circle" color="warning">
-      පාලි පෙළ සෝදුපත් තවමත් අවසන් කර නැති බව සලකන්න. අකුරු වැරදි තිබිය හැකිය. පිටු අංක මත ඔබා අදාළ scan පිටුව පරික්ෂා කර බැලිය හැක.
-      අටුවා පරිවර්තනයේද පරිවර්තන දෝෂ, මුද්‍රණ දෝෂ සහ පරිවර්තන මග හැරුණු තැන් තිබිය හැක. දෝෂ හමුවුවහොත් නිවැරදි කිරීමට අප වෙත දන්වා එවන්න. pathnirvana@gmail.com
+      පාලි පෙළ සෝදුපත් කියවීම තවමත් අවසන් කර නැති බව සලකන්න. අකුරු වැරදි තිබීමට ඉඩ ඇත. පිටු අංක මත එබීමෙන් අදාළ පරිලෝකිත පිටුව බැලීමට හැකිය.
+      අටුවා පරිවර්තනයේ ද පරිවර්තන දෝෂ, මුද්‍රණ දෝෂ හා පරිවර්තන මග හැරුණු තැන් තිබීමට හැකිය. දෝෂ හමු වුවහොත් ඒවා නිවැරදි කිරීමට අප වෙත දන්වා එවන්න. pathnirvana@gmail.com
     </v-banner>
 
     <v-container fluid>
       <v-row>
         <v-col cols="12">
           <v-card flat>
-            <v-card-title>ශ්‍රී ලංකා තිපිටක පෙළ, අටුවා සහ සිංහල පරිවර්තනය</v-card-title>
+            <v-card-title>ත්‍රිපිටක පෙළ, අටුවා සහ සිංහල පරිවර්තනය</v-card-title>
             <v-card-text>
-              තිපිටක පෙළ සහ සිංහල පරිවර්තනය බුද්ධ ජයන්ති තිපිටක ග්‍රන්ථ මාලාවෙනි. 
-              පාළි අටුවා ඡට්ඨ සංගායනාවෙන් ද එහි සිංහල පරිවර්තනය බෞද්ධ සංස්කෘත මධ්‍යස්ථාන මුද්‍රණයෙනි. 
-              මෙම වෙබ් අඩවිය සහ මෘදුකාංගය පහත ව්‍යාපෘති සහ වෙබ් අඩවි වල සාමුහික ප්‍රයත්නයක ප්‍රතිඵලයකි.
+              ත්‍රිපිටක පෙළ සහ සිංහල පරිවර්තනය බුද්ධ ජයන්ති ත්‍රිපිටක ග්‍රන්ථ මාලාවෙනි. 
+              පාළි අටුවා ඡට්ඨ සංගායනාවෙන් ද එහි සිංහල පරිවර්තනය බෞද්ධ සංස්කෘතික මධ්‍යස්ථානයේ මුද්‍රණයෙනි. 
+              මෙම වියමන අඩවිය සහ මෘදුකාංගය පහත ව්‍යාපෘති හා අඩවි වල සාමුහික ප්‍රයත්නයක ප්‍රතිඵලයකි.
```

Comment:
tipitaka not tripitaka - see above

Reply:
සමාවන්න. එම වෙනස් කිරීම ඉවත් කර ඇත.

Reply:
there is a typo here. an extra i has been added after ti

Reply:
සමාවන්න. අමතර පිල්ලම ඉවත් කළා.